docker/parsr-base/Dockerfile

```diff
@@ -22,8 +8,6 @@ RUN apt-get update && \
 WORKDIR /opt/app-root/src
 RUN chown 1001:0 /opt/app-root/src
 
-COPY --from=builder /opt/pdf2json /opt/pdf2json
-
 ENV PATH $PATH:/opt/app-root/src/node_modules/.bin:/opt/pdf2json/bin
```

Comment:
there's left a pdf2json reference in this line! 

maybe replace:
`ENV PATH $PATH:/opt/app-root/src/node_modules/.bin:/opt/pdf2json/bin`

with:
`ENV PATH $PATH:/opt/app-root/src/node_modules/.bin`

Reply:
You're right !